oktapam/resource_project_group.go

```diff
@@ -123,41 +130,35 @@ func resourceProjectGroupCreate(ctx context.Context, d *schema.ResourceData, m i
 		return diag.FromErr(err)
 	}
 
```

Comment:
Ah clever, don't need to call `SetId` because you're calling `Read` here. Got it.